README.md

```diff
@@ -0,0 +1,80 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name
+
+> Portfolio Microverse
+
+## Built With
+
+- HTML/CSS
+- NPM, Git, Github
+
+## Getting Started
+
+**This is a portfolio project that includes linters for Reporting errors in HTML, CSS and An open-source, automated tool for improving the quality of web pages, which has audits for performance, accessibility, progressive web apps, SEO and more.**
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+1. Node package Manager(NPM)
+   - To install NPM on your local machine, please visit [ NPM official website](https://nodejs.org/en/download/)
+
+- Text editor(Vscode, Atom, NotePad++, Sublime)
+
+### Setup
+
+Set up folder as:
+
+1. /Portfolio-Microverse
+   - .github/workflows - linters.yml
+   - images folder
+   - Package.json
+   - index.html
+   - style.css
+   - .stylelintrc.json
+   - .hintrc
+
+### Install
+
+Run the following commands to install dependencies
+
+```
+
+npm install --save-dev hint@7.x
+npm install --save-dev stylelint@13.x stylelint-scss@3.x stylelint-config-standard@21.x stylelint-csstree-validator@1.x
+
+```
+
+### Usage
+
+```
+
+npx hint .
+npx stylelint "**/*.{css,scss}"
+
+```
+
+### Deployment
+
```

Comment:
- [ ] `required` please make the readme descriptive in alignment with your project, see the link for readme file [requirements](https://github.com/microverseinc/readme-template/blob/master/README.md)

Reply:
README file has been updated to standard following the recommended templates.

Thanks for the review.
[@mwanawabangona](https://github.com/mwanawabangona)